geometry_msgs/TransformStamped init_guess
sensor_msgs/PointCloud2 point_cloud2
uint32 seq

